enterprise/frontend/src/embedding-sdk/lib/theme/default-component-theme.ts

```diff
@@ -85,6 +86,9 @@ export const DEFAULT_METABASE_COMPONENT_THEME: MetabaseComponentTheme = {
       label: { fontSize: FONT_SIZES.goalLabel.px },
     },
   },
+  popover: {
```

Comment:
I was going to call this `clickActions.popover`, but it turns out `TippyPopover` is used in other components too (e.g. `LinkViz`, `DateWidget`), so that would be a misleading name too...

We could add the `zIndex` here to the Mantine-based Popover as well, but I am afraid that would break other things.

Reply:
> add this zIndex value as the defaults of Mantine-based Popover as well
without this it's a bit weird though, how would the user know what popovers are going to be affected?
Would it be possible to find a sensible default value instead?
If I understood the problem correctly, it's when the user puts a question in a modal with higher zIndex than the one of the popover, maybe if we make our zIndex higher we can avoid having to add this extra "field" to the public api of the sdk which would only work on some popovers

Reply:
@npretto yeah, it is quite confusing because of the fact we have multiple popover components... The problem with setting a single value is that we don't know what the z-index of the user's modal is going to be, as that's completely dependent on their application. (some libraries even use arbitrary default values like "200")

Reply:
@npretto I added the z-index to the Mantine-based popover when in embedding sdk. This should now apply to all popovers 🙏🏻